pyiron_base/job/script.py

```diff
@@ -31,6 +31,39 @@ class ScriptJob(GenericJob):
         project (ProjectHDFio): ProjectHDFio instance which points to the HDF5 file the job is stored in
         job_name (str): name of the job, which has to be unique within the project
 
+    Example:
+        To submit a notebook as a ScriptJob job to the pyiron job management system,
+
+            1. Create the notebook to be submitted, for ex. 'example.ipynb', and save it -- Can contain any code like:
+                ```
+                from pyiron import Project
+                pr = Project('.')
+                job = pr.create_job(pr.job_type.Lammps, 'job')
+                job.structure = pr.create_ase_bulk('Fe')
+                job.run()
+                print(job.status)
+                print(job.output.energy_pot[-1])
+                ```
+
+            2. Create the notebook which submits the notebook created in 1, for ex. 'submit_example_job.ipynb',
+                which can have the following code:
+                ```
+                from pyiron import Project
+                pr = Project('scriptjob_example')  # save the ScriptJob in the 'scriptjob_example' project
+                scriptjob = pr.create_job(pr.job_type.ScriptJob, 'scriptjob')  # create a ScriptJob named 'scriptjob'
+                scriptjob.script_path = 'example.ipynb'  # specify the PATH to the notebook you want to submit.
+                                                         # In this example case, 'example.ipynb' is in the same
+                                                         # directory as 'submit_example_job.ipynb'
+
+                # to submit the notebook to a queueing system
+                scriptjob.server.queue = 'cmfe'  # specify the queue to which the ScriptJob job is to be submitted
+                scriptjob.server.cores = 4  # specify the number of cores to be used
+                scriptjob.server.run_time = 120  # specify the runtime limit for the ScriptJob job in seconds
+
+                # run the ScriptJob job
+                scriptjob.run()
+                ```
```

Comment:
Generally I find it also helpful to know how to retrieve or analyse output data, but I wouldn't insist on it because it might be trivial if the user looks at `job_table()`.

Reply:
I will add a bit more documentation on how to check if the notebook is submitted/running/finished.

Reply:
Agree, please add how to retrieve the results from the notebooks.  If I remember an issue from last year, it's not quite intuitive.

Reply:
I usually do it by reloading the job executed within the submitted notebook. Is there another way of doing this?

Reply:
Maybe you can mention in the notebook to submit (i.e. for the LAMMPS job) that the user can store data in `job['user']`, by something like:

```
job['user/output_value'] = some_output
```

And if `Project(default_working_directory=True)` is used, the job can be loaded from the submitter notebook via the job name just like any other job in the same notebook. If a different project name is used, I don't know how it works (that's something I never do).

Reply:
It so happens that ScriptJob does not care in which project the job is saved, as long as it has been run in the submitted notebook (here, 'example.ipynb'). Which I think is really nice! :)